_posts/2020-01-01-2019-summary.md

```diff
@@ -0,0 +1,230 @@
+  in cost based optimizer.
+* Improved accounting and reporting of physical and network data read or transmitted during query processing.    
+
+## Performance
+
+* [10x performance improvement for `UNNEST`]({% post_url 2019-08-23-unnest-operator-performance-enhancements %}).
+* 2-7x improvement in performance of [ORC decoders]({% post_url 2019-04-23-even-faster-orc %}), resulting in a 
+  10% global CPU improvement for the TPC-DS benchmark.
+* Improvements when reading small Parquet files, files with large number of columns, or files with small row
+  groups.
+* Support for new ORC bloom filters.
+* Remove [redundant `ORDER BY`]({% post_url 2019-06-03-redundant-order-by %}) clauses.
+* Improvements for [`IN` / `NOT-IN`]({% post_url 2019-06-03-redundant-order-by %}) with subquery expressions (i.e., semijoin).
+* Huge performance improvements when [reading from `information_schema`](https://github.com/prestosql/presto/pull/1329).
+* Reduce query latency and Hive Metastore load.
+* Improve metadata handling during planning. This can result in dramatic improvements in latency, 
+  especially for connectors such as MySQL, PostgreSQL, Redshift, SQL Server, etc. Some queries like 
+  `SHOW SCHEMAS`/`SHOW TABLES` that could take several minutes to complete now finish in a few seconds.
+* Improved stability and performance when spilling is enabled.  
+
+## Functions
+
+* [`combinations`](/docs/current/functions/array.html#combinations)
+* [`format`](/docs/current/functions/conversion.html#format)
+* [`UUID` type](/docs/current/functions/uuid.html) and related functions.
+* [`all_match`](/docs/current/functions/array.html#all_match),
+  [`any_match`](/docs/current/functions/array.html#any_match) and 
+  [`none_match`](/docs/current/functions/array.html#none_match).
+* Support for higher-order [`reduce_agg`](/docs/current/functions/aggregate.html#reduce_agg) 
+  function in aggregation queries.
+* New datetime functions: [`last_day_of_month`](/docs/current/functions/datetime.html#last_day_of_month),
+  [`at_timezone`](/docs/current/functions/datetime.html#at_timezone) and 
+  [`with_timezone`](/docs/current/functions/datetime.html#with_timezone).
+
+## Security
+
+* [Role-based access control](/docs/current/sql/create-role.html) and related commands.
+* [INVOKER security mode](/docs/current/sql/create-view.html#security) for views, which allows views to be run using the permissions of the 
+  current user.
+* Prevent replay attacks and result hijacking in client APIs.
+* JWT-based [internal communication](/docs/current/security/internal-communication.html#internal-authentication) authentication,
+  which obsoletes the need to use Kerberos or certificates and greatly simplifies secure setups.
+* Credential passthrough, which allows Presto to authenticate with the underlying data source with 
+  credentials provided by the user running a query. We found this especially useful when dealing with
```

Comment:
I'd remove "we found" to better match the style of the rest of the post and replace with

> This is especially useful when ...